test/integration/setup.js

```diff
@@ -123,8 +123,27 @@ const authProxy = (token) => ({
     return (...args) => method.apply(target, args).set('Authorization', `Bearer ${token}`);
   }
 });
+
+// Credit: @alxndrsn
+// Add User-Agent header to all requests.  This was previously a default in
+// superagent, but was removed in 5.1.0.
+// See: https://github.com/ladjs/superagent/pull/1495
+const uaProxy = {
```

Comment:
Do we need this proxy for this PR, or could we wait for #899 to be merged?

Reply:
we can wait